### ISSUE TITLE
Admin app with adding new projects

### ISSUE THREAD

**Issue by davydovanton** [hanami/contributors#39]
We can split this task to PRs:

- [ ] generate new `admin` application;
- [ ] protect `admin` app by rack simple auth (password and user setup by ENV variables, default - `hanami/hanami';
- [ ] add page with all projects in db;
- [ ] add form for adding a new project (with spawning sidekiq worker for export all commits)

**Comment by Bounga** [comment]
Gonna work on adding a page listing all projects available in db tonight or at the latest this week-end if it's ok for you.

**Comment by davydovanton** [comment]
yep, I'll be happy to see PR ;)

**Comment by GustavoCaso** [comment]
If no one is working on the list item from the list I would like to work on it 🎉

**Comment by Bounga** [comment]
@GustavoCaso Are you talking about the "add page with all projects in db;" item? If so it's already done.

I've started working on the last remaining task which is a form to add a project and spawning a background task to retrieve associated commits. If you're really interested in doing it I can stop my work and left it to you.

**Comment by GustavoCaso** [comment]
Hi @Bounga please do not stop your work on that one. I would love to see how you solve it. 🎉 

Thanks for the quick response.

**Comment by Bounga** [comment]
@GustavoCaso You're welcome. Hope to deliver this feature tonight. Feel free to comment on it when it's available.

**Comment by davydovanton** [comment]
@GustavoCaso if you want - I have other ideas for this project, we can start work on it 👍

**Comment by GustavoCaso** [comment]
@davydovanton I would love to hear them ❤️💛💚. I have been meaning to try hanami and I think these is the best way

**Comment by Bounga** [comment]
@GustavoCaso I also think it's one a the best way to start with Hanami. And it's useful to other people. That's why I'm contributing. If you want to peer on some feature I'd really like to.

@davydovanton I'd love to hear about it too. I saw an idea about creating a variable for the organization and I think it's a freaking good idea. I think I'll have some free time in the next weeks to dedicate to this project. If I can help, I will.

**Comment by Bounga** [comment]
I've started working on the remaining task but I don't have a lot of free time these days. It's going to be easier for me to finish this next week. Here's what I've done so far https://github.com/Bounga/contributors/tree/add-new-project-from-admin

**Comment by davydovanton** [comment]
@Bounga thanks for share your status ❤️ 
don't worry about it :)

**Comment by nagano564** [comment]
is someone still working on this??

**Comment by Bounga** [comment]
@nagano564 Yes I’m still working on it. I do have something working actually that you can see in my branch linked above. I still want to add some missing tests and refactor duplicated code. I think it’ll be done by this week.
